docs/advanced-recipes/atom-creators.mdx

```diff
@@ -355,4 +355,105 @@ const ListOfThings = () => {
 }
 ```
 
-<CodeSandbox id="ugkzm0" />
```

Comment:
We should not delete this, right?

Reply:
Sorry, was cutting the snippet for my code sandbox section and forgot to add it back. Amended here: [f5e72d3](https://github.com/pmndrs/jotai/pull/1338/commits/f5e72d36ee9820191a02e1ec40c774c55795ed37)